cfn-resources/project-ip-access-list/README.md

```diff
@@ -2,10 +2,7 @@
 
 
 ## Description
-The Access List endpoint manages a Atlas project's IP Access List.
-
-The Access List endpoint supports creating temporary Access List entries that automatically expire within a user-configurable 7-day period.
-
+Returns, adds, edits, and removes network access limits to database deployments in MongoDB Cloud.
 
 ## Attributes
 
```

Comment:
Parameters section below is stale, try to copy as much as you can from [Open API docs](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Project-IP-Access-List/operation/returnProjectIpAccessList). For example, delete reference to API keys and update ProjectId with. 

ProjectId (required) - Unique 24-hexadecimal digit string that identifies your project.

Reply:
Docs is updated with latest OpenAPI spec